Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-rails (3.8.2)
+    rspec-rails (3.8.3)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       railties (>= 3.0)
       rspec-core (~> 3.8.0)
       rspec-expectations (~> 3.8.0)
       rspec-mocks (~> 3.8.0)
       rspec-support (~> 3.8.0)
-    rspec-support (3.8.2)
-    rubocop (0.74.0)
+    rspec-support (3.8.3)
```

Comment:
### [rspec-support](https://github.com/rspec/rspec-support)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.8.2 → 3.8.3

[_[change-log](https://github.com/rspec/rspec-support/blob/v3.8.3/Changelog.md), [source-code](https://github.com/rspec/rspec-support)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-support/compare/v3.8.2...v3.8.3).

These are the individual commits:
- (9601929) [Merge pull request #366 from edzhelyov/recognize-hash-as-opt…](https://github.com/rspec/rspec-support/commit/9601929142d19f4bce955798a910b9fd3912b18f)
- (3b0fd00) [Fix typo :)](https://github.com/rspec/rspec-support/commit/3b0fd000a19ca29438b7c1b916540a35310cff0e)
- (4f35c62) [Merge pull request #371 from olleolleolle/update-travis-buil…](https://github.com/rspec/rspec-support/commit/4f35c621452771bb51d94b44a6245c2f533366e2)
- (7a3b7d6) [Merge pull request #378 from rspec/escape-slash-r](https://github.com/rspec/rspec-support/commit/7a3b7d6940a2ddd3bcf6ba827742014a347d1acd)
- (ad3afce) [Merge pull request #380 from rspec/update-travis-build-scrip…](https://github.com/rspec/rspec-support/commit/ad3afce2a5cd495bc29ed8d78a7c49a665c880f9)
- (197c438) [Merge pull request #381 from rspec/update-travis-build-scrip…](https://github.com/rspec/rspec-support/commit/197c438899e005ec6227bdce2ad4ada651cbb9f6)
- (fd7d2f9) [Synchronise appveyor.yml with master](https://github.com/rspec/rspec-support/commit/fd7d2f95334f86bdb4bc6ba054fd3dfb784a851e)
- (78657b0) [Release 3.8.3](https://github.com/rspec/rspec-support/commit/78657b0f7b50cf4b451e96a7b3c4e3179bacb328)


</details>